package.json

```
@@ -14,9 +14,9 @@
     "format": "biome format --write",
     "db:up": "docker compose up -d",
     "db:down": "docker compose down",
-    "db:push": "bunx drizzle-kit push --force",
-    "db:generate": "bunx drizzle-kit generate",
-    "db:migrate": "bunx drizzle-kit migrate",
+    "db:push": "bun run drizzle-kit push --force",
+    "db:generate": "bun run drizzle-kit generate",
+    "db:migrate": "bun run drizzle-kit migrate",
```

Redundant. By running `bun run db:push`, it already converts to `bun run drizzle-kit push --force`, so you can remove this. I've tried and it works on my end, but try it on your end too.
```suggestion
    "db:push": "drizzle-kit push --force",
    "db:generate": "drizzle-kit generate",
    "db:migrate": "drizzle-kit migrate",
```